steely/plugins/b.py

```diff
@@ -0,0 +1,37 @@
+COMMAND = '.b'
+__author__ = ('byxor',)
+
+
+def main(bot, author_id, message, thread_id, thread_type, **kwargs):
+    last_message = lambda: bot.fetchThreadMessages(thread_id=thread_id, limit=2)[1]
+    send = lambda message: bot.sendMessage(message, thread_id=thread_id, thread_type=thread_type)
+
+    send(encode(last_message()))
+
+    
+REPLACEABLE = [character for character in 'PpBbGg']
+REPLACEMENT = '🅱️'
+    
+
+def encode(string):
+    for sequence in REPLACEABLE:
```

Comment:
Wouldn't using string.translate be easier to read? ie.
```
return string.translate(REPLACEABLE, REPLACEMENT*len(REPLACEABLE))
```

Reply:
I want to support sequences other than single characters later. E.g. maybe someone will want to change "ck" to :b: or something. You're right though.